### ISSUE TITLE
Prefill event location with room name(s)

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/calendar#3420]

**Comment by tcitworld** [comment]
Also in rfc9073 which just dropped there's a new `VRESOURCE` component that can be used, where `RESOURCE-TYPE` can be set to `ROOM`.

**Comment by ChristophWurst** [comment]
@tcitworld this is really cool. I guess it will take some time until client applications pick this up. Would it make sense to write to vresource *and* lcoation for the time being?

**Comment by tcitworld** [comment]
Yeah, supporting `VRESOURCE` will need work on `ical.js` and/or `calendarjs` anyway.
